### PR TITLE
Redesign of Server Connect. 

### PR DESCRIPTION
**Pull request by piperchester**
- Rearchitecting in Storyboard
- Implementing favorites addition/deletion
- Repositioning of QR Scan button
- Fixing direct connect segue

### CODE REVIEW

File: AccessLecture/ConnectionViewController.h [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 @end
 
-@interface ConnectionViewController : UIViewController
+@interface ConnectionViewController : UIViewController <UITableViewDataSource, UITableViewDelegate>
```
Comment (7imbrook) [inline AccessLecture/ConnectionViewController.h:25]: New line after @interface, see coding guidelines

Comment (piperchester) [inline AccessLecture/ConnectionViewController.h:25]: :ok_hand: - On it.

**Comment by 7imbrook** [comment]
Merge Integration into your branch before submitting pull requests @piperchester

File: AccessLecture/ConnectionViewController.m [hunk 2]
```diff
@@ -53,11 +43,34 @@ - (void)viewDidLoad
...
-        [_scanButtonView removeFromSuperview];  // Hiding QR Scan! Button and QR image.
         [_previewView setBackgroundColor:[UIColor redColor]];
     }
     
-    isScanning = NO;
+    // Initializing Lecture and Server arays.
+    if (!lectureFavorites) lectureFavorites = [NSMutableArray new];
+    if (!serverFavorites) serverFavorites = [NSMutableArray new];
+    
+    // Adding Gestures.
```
Comment (7imbrook) [inline AccessLecture/ConnectionViewController.m:53]: Table view controller should handle this for you.

**Pull request merged**